endpoints/base

```diff
@@ -25,7 +25,8 @@ user=""
 engine_script_start_timeout=""
 max_sample_failures=""
 total_cpu_partitions=0
-export ssh_opts="-q -o StrictHostKeyChecking=no -o PasswordAuthentication=no"
+export ssh_opts="-q -o StrictHostKeyChecking=no -o PasswordAuthentication=no -o ControlMaster=auto -o ControlPath=~/.ssh/sockets/%r@%h-%p -o ControlPersist=60"
```

Comment:
I have no idea what this does :)
 %r@%h-%p

Reply:
IIRC thiis creates or uses a socket with the filename being of the form `<username>@<hostname>-<port>` so that repeated connections to a host with the same username on the same port can reuse the existing connection that created the socket.